woocommerce/woocommerce-android
# Feature/10224 theme picker tracking

Closes: #10382


### Description

Apply the following tracking plan for theme picker feature:

![Screenshot 2023-12-13 at 16 26 44](https://github.com/woocommerce/woocommerce-android/assets/2663464/56faf623-c1b0-4f08-9d56-143383777605)


### Testing instructions

See the above table for reference. Follow the steps below and check for the tracked events in the logcat

**From settings**
1. Open Theme picker screen from settings: `Tracked: theme_picker_screen_displayed, Properties: {"source":"settings"`
2. Select any theme from the picker to open the preview: `Tracked: theme_picker_theme_selected, Properties: {"theme":"Attar"`
3. Preview is opened: `Tracked: theme_preview_screen_displayed`
4. Change theme page: `theme_preview_page_selected, Properties: {"page":"My Account"`
5. Change device form factor layout: `Tracked: theme_preview_layout_selected, Properties: {"layout":"mobile"`
6. Disable the internet and tap on "Use this theme": `Tracked: theme_preview_start_with_theme_button_tapped, Properties: {"theme":"Tsubaki"` and  `theme_installation_failed, Properties: {"theme":"Tsubaki"`
7. Enable internet connection again and tap on "Use this theme": `Tracked: theme_installation_completed, Properties: {"theme":"Tsubaki"`


**From store creation**
1. Open theme picker from store creation and check: `theme_picker_screen_displayed, Properties: {"source":"profiler"` 
**Note:** About the `source` value for this case, ' profiler`, do you think it might make more sense to name it `store_creation`?  Imo the theme picker is not a profiler question/step. It is just another screen in the store creation flow. Wdyt @hafizrahman ?
